docs/apm/correlations.asciidoc

```diff
@@ -76,16 +83,40 @@ existed in 100% of failed transactions between 6:00 and 10:30.
 [role="screenshot"]
 image::apm/images/error-rate-hover.png[Correlations errors hover effect]
 
-Selecting the `+` filter creates a new query in the APM app for transactions with
+Select the `+` filter to create a new query in the APM app for transactions with
 `url.original: http://localhost:3100...`. With the "noise" now filtered out,
 you can begin viewing sample traces to continue your investigation.
 
+As you sift through erroneous transactions, you'll likely notice other interesting attributes.
+Return to the correlations fly-out and select *Customize fields* to search on these new attributes.
+You may need to do this a few time–each time filtering out more and more noise and bringing you
+closer to a diagnosis.
+
 [discrete]
-[[correlations-custom-fields]]
+[[correlations-customize-fields]]
 ====  Customize fields
 
 Correlations are only as good as the data they're searching for.
-By default, a handful of potentially useful fields are selected, like `lables`, `service.version`, and `host.ip`.
-You can remove and add fields to this list under the **Customize fields** dropdown.
+By default, a handful of attributes commonly known to cause performance issues are included.
+During the course of an investigation however, you may to need to add and remove fields from
+this list multiple times as you narrow in on a diagnosis.
+
+Add and remove fields under the **Customize fields** dropdown.
+The following fields are selected by default:
+
+**Frontend (RUM) agent:**
+
+* `labels.*`
+* `user.*`
+* `user_agent.name`
+* `user_agent.os.name`
+* `url.original`
+
+**Backend agents:**
+
+* `labels.*`
+* `host.ip`
+* `service.node.name`
+* `service.version`
```

Comment:
Based on https://github.com/elastic/kibana/blame/21587dc79efe99294dadb90113ae998d1f8e3693/x-pack/plugins/apm/public/components/app/correlations/use_field_names.ts#L61-L73.

To be verified by @ogupte

Reply:
correct, for the open ended one with the asterisk, it just grabs the first 6 of those fields just to keep it manageable.

Reply:
Ahh, duh `.slice(0,6)`